docs-sources/includes/_Archivist.md

```diff
@@ -35,6 +50,14 @@ a short configuration description for each available vault backends
 alongside a list of supported operations for that backend.
 </aside>
 
+
+### List, read and write order
+
+In `./config/default.yaml`, listOrder, readOrder, and writeOrder properties **have to** be defined to specify the order when reading or writting data.<br>
```

Comment:
This can be specified and customized in any configuration file - it is not limited to default.